app/renderer/js/utils/domain-util.js

```diff
@@ -106,7 +108,19 @@ class DomainUtil {
 
 		domain = this.formatUrl(domain);
 
-		const checkDomain = domain + '/static/audio/zulip.ogg';
+		const certificate = CertificateUtil.getCertificate(encodeURIComponent(domain));
+		let certificateLocation = '';
+
+		if (certificate) {
+			// To handle case where certificate has been moved from the location in certificates.json
+			try {
+				certificateLocation = fs.readFileSync(certificate);
+			} catch (err) {
+				console.log(err);
+			}
+		}
+
+		const checkDomain = (certificateLocation) ? ({url: domain + '/static/audio/zulip.ogg', ca: certificateLocation}) : domain + '/static/audio/zulip.ogg';
```

Comment:
Can you add a comment just above this, explaining a bit about what's going on?

Reply:
I'll add one in the next commit push.